charts/stardog/templates/post-install-job.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "stardog.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    helm.sh/chart: {{ include "stardog.chart" . }}
+    app.kubernetes.io/name: {{ .Chart.Name }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+  annotations:
+    "helm.sh/hook": post-install
```

Comment:
I was reading the docs on the hook at it's not clear to me if the helm actually then waits on the rest of the resources reaching ready pushing  the resources that are part of the post install step. Like I'm wondering if you have to maintain a connection to the K8S API while the resources reach this state or if this blocking is actually somehow done on the backend? Do you know Paul?

Reply:
Not sure I follow what you're asking exactly, but I don't think it does block on all the other resources -- it simply waits for helm install to complete (which returns quickly). The job to change the password will fire before all 3 Stardogs are ready, and it will periodically hit the endpoint trying to change the password until it succeeds from my observations.